tests/test_output.py

```diff
@@ -94,3 +97,48 @@ def test_print_hw(hq_env: HqEnv):
     output = hq_json_wrapper(hq_env, ["--output-type=json", "worker", "hwdetect"])
     assert isinstance(output, dict)
     assert "cpus" in output.keys()
+
+
+'''
+Quiet flag tests
+'''
+
+
+def test_print_worker_list(hq_env: HqEnv):
```

Comment:
`test_print_worker_list` and `test_print_job_list` have a duplicated name. Please add `_quiet` suffix to their names.